app/src/main/java/com/eveningoutpost/dexdrip/Services/DexCollectionService.java

```diff
@@ -1317,6 +1322,14 @@ public void run() {
             }
         }
 
+        // blueReader
+        ActiveBluetoothDevice activeBluetoothDevice = ActiveBluetoothDevice.first();
```

Comment:
Can you use the static method `isblueReader()` here to avoid duplication?

Reply:
Correct, seen that also too late after pull, will be changed in next commit.